controllers/message.go

```diff
@@ -180,6 +180,37 @@ func (c *ApiController) AddMessage() {
 		return
 	}
 
+	id := util.GetIdFromOwnerAndName(message.Owner, message.Name)
```

Comment:
Define `func DeleteAllLaterMessages(messageId string) error` in object pkg to put your code